### PR TITLE
caffe2fluid: support deconvolution convertion

### PR DESCRIPTION
**Pull request by walloollaw**
updated features:
1, support deconvolution convertion
2, support tanh layer convertion
3, support slice layer convertion indirectly
4, add missed files for SSD model convertion

### CODE REVIEW

File: fluid/image_classification/caffe2fluid/kaffe/custom_layers/priorbox.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        output (variable): output variable for this layer
+    """
+    import paddle.fluid as fluid
+
+    assert len(inputs) == 2, "invalid inputs for Priorbox[%s]" % (name)
+    input = inputs[0]
+    image = inputs[1]
+    box, variance_ = fluid.layers.prior_box(input, image, min_size, max_size,
+                                            aspect_ratio, variance, flip, clip,
+                                            (step, step), offset)
```
Comment (qingqing01) [inline fluid/image_classification/caffe2fluid/kaffe/custom_layers/priorbox.py]: The PR https://github.com/PaddlePaddle/Paddle/pull/12032  has been merged, please update.

File: fluid/image_classification/caffe2fluid/kaffe/custom_layers/roipooling.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    base_fea_shape = input_shapes[0]
+    rois_shape = input_shapes[1]
+    output_shape = base_fea_shape
+    output_shape[0] = rois_shape[0]
+    output_shape[2] = pooled_h
+    output_shape[3] = pooled_w
+    return output_shape
+
+
+def ROIPooling_layer(inputs, name, pooled_h, pooled_w, spatial_scale):
```
Comment (qingqing01) [inline fluid/image_classification/caffe2fluid/kaffe/custom_layers/roipooling.py]: 其他layer名字看着都是小写，这里是大写，要改成小写吗?

File: fluid/image_classification/caffe2fluid/kaffe/custom_layers/priorbox.py (hunk 1 above)
Comment (walloollaw) [reply fluid/image_classification/caffe2fluid/kaffe/custom_layers/priorbox.py]: done

File: fluid/image_classification/caffe2fluid/kaffe/custom_layers/roipooling.py (hunk 2 above)
Comment (walloollaw) [reply fluid/image_classification/caffe2fluid/kaffe/custom_layers/roipooling.py]: 这个是由用户实现的，大家一般把这个名字写成layer的type对应的字符串，不过为了风格统一，我改过来了。

**Review by qingqing01 — APPROVED** [review]

**Pull request merged**